app/react/Auth/components/NeedAuthorization.tsx

```diff
@@ -18,7 +18,7 @@ const checkWritePermissions = (entities: EntitySchema[] = [], user?: any) => {
   while (granted && i < entities.length) {
```

Comment:
At first I was going to sugest change this to a reduce, but I guess while will end early if you don't have permission for one entity